### PR TITLE
Covenant Training Pistol

### PR DESCRIPTION
**Pull request by ghost**
Adds a recoloured plasma pistol with TTR code for the purposes of training.

The update pistols.dm commit doesn't actually do anything, thankfully. It's overwritten by the proper Covenant Training Pistol commit.

Please God let this be the last time.

### CODE REVIEW

File: code/modules/halo/weapons/covenant/pistols.dm [hunk 1]
```diff
@@ -58,8 +58,12 @@
 	desc = "A dual funtionality pistol: It fires bolts of plasma, and when overcharged is capable of emitting a small emp burst at the point of impact. This one appears to be disabled"
 	max_shots = 0
 
-/obj/item/weapon/gun/energy/plasmapistol/disabled/attack_self(var/mob/user)
```
Comment (X0-11) [inline code/modules/halo/weapons/covenant/pistols.dm]: This shouldn't be removed.

Comment (ghost) [reply code/modules/halo/weapons/covenant/pistols.dm]: handled with the most recent updated one

And also, sorry that I haven't been naming my commits, it slipped my mind up to now

File: code/modules/halo/weapons/covenant/pistols.dm [hunk 2]
```diff
@@ -59,7 +59,13 @@
 	max_shots = 0
 
 /obj/item/weapon/gun/energy/plasmapistol/disabled/attack_self(var/mob/user)
-	return
```
Comment (caelaislinn) [inline code/modules/halo/weapons/covenant/pistols.dm:62]: Although removing this line doesn't change anything, it's better not to make unnecessary changes.

File: code/modules/halo/weapons/covenant/pistols.dm [hunk 3]
```diff
@@ -59,7 +59,13 @@
...
+
+/obj/item/weapon/gun/energy/plasmapistol/trainingpistol
+	name = "Type-25B Directed Energy Pistol"
```
Comment (caelaislinn) [inline code/modules/halo/weapons/covenant/pistols.dm]: It might be a good idea to give this a name like "Type-25B Directed Energy (Training) Pistol" so it doesn't share the same name as the actual plasma pistol

**Review by caelaislinn — APPROVED** [review]
I have a few minor code suggestions but this is ready to merge regardless

File: code/modules/halo/weapons/covenant/pistols.dm (hunk 3 above)
Comment (ghost) [reply code/modules/halo/weapons/covenant/pistols.dm]: Done and done.

**Review by caelaislinn — APPROVED** [review]

**Comment by caelaislinn** [comment]
Closing and reopening to trigger a new travis run

**Comment by caelaislinn** [comment]
This can be merged, the Travis failure is due to a Geminus issue introduced in #1302 and I have a fix waiting in #1303
